pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.black]
```

Comment:
What is this .toml file for? I don't know the format ;) Is it just for black formatting?

Reply:
Toml is just another markup language, the pyproject. toml file is a project file that some plugins like black use for configuration, in some cases I've seen it used with setup.py too. I've not used black before, normally used autopep8 but I quite like black! Reading the docs it looks for pyproject.toml for configuration.